src/Microsoft.PowerShell.SHiPS/Microsoft.PowerShell.SHiPS.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net46;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>net471;netcoreapp2.0</TargetFrameworks>
```

Comment:
Are you sure that you want or need to make .Net 4.7.1 the default version? Aiming for .Net 4.5.2 or PowerShell Standard 5.1 will make it more portable won't it?

Reply:
We have been using .Net4.6. No issues. No complains. There are some APIs added in 4.7.1 such as RuntimeInformation.IsOSPlatform() which I can use  for detecting OS platform. Otherwise, I have to use PInvoke or [System.IO.Path]::DirectorySeparatorChar == '\\' for windows.  Any suggestions?

Reply:
[.Net Standard 2.0](https://blogs.msdn.microsoft.com/dotnet/2017/08/14/announcing-net-standard-2-0/) should be the target. It uses 4.6.1 and is compatible with PowerShell Core so it can be a "build once, run anywhere" app.

Reply:
I have not moved to .netstandard 2.0 yet. I am still doing coreclr and fullclr. The major concern was to support downlevel OSs such as Win7.